lms/envs/test.py

```diff
@@ -582,3 +582,7 @@
 COURSE_CATALOG_API_URL = 'https://catalog.example.com/api/v1'
 
 COMPREHENSIVE_THEME_DIRS = [REPO_ROOT / "themes", REPO_ROOT / "common/test"]
+
+ENV_TOKENS = {
+    'LMS_ROOT_URL': 'https://localhost:8000',
+}
```

Comment:
`LMS_ROOT_URL` should be a top-level setting here. 

`ENV_TOKENS` is only for passing in settings from JSON files, and shouldn't be accessed outside of the settings files.